web/beacon-app/src/constants/app.ts

```diff
@@ -8,7 +8,7 @@ export const APP_ROUTE = {
   DASHBOARD: '/dashboard',
   TENANTS: '/tenant',
   APIKEYS: '/apikeys',
-  PROJECTS: '/projects',
+  PROJECTS: '/{:tenantID}/projects',
```

Comment:
I think it would be better to create a new route for the projects list.
i use this to get the project detail where the route is `projects/:projectID`. i think you can leave it as it was before.

Reply:
I'm pretty sure the route will need to be `:tenantID/projects for List and Create. When I checked CreateUserResources it uses the TenantProjectCreate handler. I can create a separate route, but it should include the tenant id.